tests/test_email.py

```diff
@@ -64,3 +64,66 @@ def test_fetch_emails(mocker):
     assert feed2.title == "Another List"
     assert feed2.is_mailing_list is True
     assert len(article.get_by_feed(feed2.id)) == 1
+
+
+def test_email_subject_lacks_sanitization() -> None:
+    """Test that email subjects are processed without sanitization.
+    
+    This demonstrates the input validation vulnerability where malicious
+    content in email subjects is not sanitized.
+    """
+    from src.email import _extract_email_subject
+    from email.message import EmailMessage
+    
+    # Create email with malicious subject content
+    msg = EmailMessage()
+    malicious_subjects = [
+        "<script>alert('xss')</script>",  # XSS payload
+        "'; DROP TABLE articles; --",  # SQL injection attempt
+        "{{7*7}}",  # Template injection attempt
+        "\x00\x01\x02malicious",  # Binary/control characters
+    ]
+    
+    for malicious_subject in malicious_subjects:
+        msg["subject"] = malicious_subject
+        
+        # Extract subject without sanitization
+        extracted = _extract_email_subject(msg)
+        
+        # The vulnerability: malicious content is preserved without sanitization
+        assert malicious_subject in extracted
+        
+        # Demonstrate specific threats
+        if "<script>" in malicious_subject:
+            assert "<script>" in extracted  # XSS payload preserved
```

Comment:
please test that the vulnerability is solved: `not in` 

also for the other tests and asserts

Reply:
and mark the test as expected to fail for now